### ISSUE TITLE
HTML Formatter issue

### ISSUE THREAD

**Issue by nazarkryp** [Microsoft/vscode#59324]
- VSCode Version: 1.27.2
- OS Version: Windows 10 

**Code is not formatted:**
![image](https://user-images.githubusercontent.com/23178377/46009343-e3f04780-c0c7-11e8-8656-bd781d5925c4.png)

**code --disable-extensions**
After using command **code --disable-extensions** it seems to be working fine. BUT the version of vscode is downgraded to from 1.27.x to 1.26.1

**Comment by aeschli** [comment]
I tried with the following example in 1.28, all extensions disabled, no custom html formatter settings
```
<button>
	<div>
		<span>some text</span>
		<img src="asads" alt="stupid">
	        </div>
</button>
```
and this gets nicly formatted to
```
<button>
	<div>
		<span>some text</span>
		<img src="asads" alt="stupid">
	</div>
</button>
```

Do you use extension that replaces the HTML formatter? Please file the issue against that extension.

**Comment by nazarkryp** [comment]
@aeschli code that you were able to format is my whole sample. i've created *.html file with code below and it is still reproducing. Dispite the fact that I've disabled all extensions, possibly one of them still has impact on that. Potentially it could be Beautity - the only extension that has something to do with  formatting. Maybe some leftovers after I've removed it.

```
<button>
	<div>
		<span>some text</span>
		<img src="asads" alt="stupid">
	        </div>
</button>
```
